docs/send-data/reference-information/time-reference.md

```diff
@@ -285,4 +285,4 @@ Search Time Ranges can also search all data with any and all timestamps. For det
 
 ## Date format
 
-If the browser used to access Sumo Logic is in a location that uses the day/month/year format instead of month/day/year, dates are presented in that format.
+If the browser used to access Sumo Logic is in a location that uses the day/month/year format instead of month/day/year, dates are presented in that format.
```

Comment:
```suggestion
If the browser used to access Sumo Logic is in a location that uses the day/month/year format instead of month/day/year, dates are presented in that format.
```